controllers/user.js

```diff
@@ -189,8 +191,31 @@ const deleteUser = async (req, res) => {
 
         for (let i = 0; i < user.workouts.length; i++) {
             const workout = await Workout.findById(user.workouts[i]);
+
             if (workout.user.toString() === userID) {
```

Comment:
this are not all the possiblities that can happen to a workout after user is deleted.
- user is the owner
    - company array is empty - delete workout
    - company array is not empty
        - workout has not happend yet - take first person from company and make them the owner
        - workout has happend - delete user from the owner, leave company as is
- user is not the owner
    - there is no owner and user is the only company - delete workout
    - there is owner - delete user id from company / access

Reply:
You said that we store all workouts, so I don't get it.

Reply:
we do, if there is a user that could access those workouts.